### ISSUE TITLE
Warnings not being written to _GOLLUM_ stream

### ISSUE THREAD

**Issue by MaxBosse** [trivago/gollum#3]
Gollum v0.3.2, FreeBSD 10.1 (vm)

I am currently trying to enable logging gollum errors and warnings into a logfile.

My test-config is the following:
```yaml
- "consumer.Console":
    Enable: true
    Stream: "test_stream"

- "producer.File":
    File: "/appdata/gollum/stream/test_stream.log"
    Stream:
      - "test_stream"
    Formatter: "format.Envelope"
    Rotate: true
    RotateSizeMB: 512
    RotateAt: "00:00"
    Compress: true
# Trying to cause a warning with the following line
    foo: bar

- "producer.File":
    File: "/var/log/gollum.log"
    Stream:
      - "_GOLLUM_"
    Formatter: "format.Envelope"
    Rotate: true
    RotateSizeMB: 512
    RotateAt: "00:00"
    Compress: true
```

Using loglevel 1 will cause the warning to be printed onto my console and leave me with an empty/non existing logfile.
```
Warning: pluginconfig.go:58: Unknown configuration key in producer.File: foo
```

Using loglevel 3 will still cause the warning to be printed onto my console, but log the startup and shutdown messages into my log-file:

Logfile:
```
We be nice to them, if they be nice to us. (startup)
Master betrayed us. Wicked. Tricksy, False. (signal)
Filthy little hobbites. They stole it from us. (shutdown)
```

Console:
```
Warning: pluginconfig.go:58: Unknown configuration key in producer.File: foo
```

Is there a possibility to write all errors/warnings into a logfile without having to redirect stderr via the console-command?

**Comment by arnecls** [comment]
Configuration warnings are a little bit special as they are written before any stream or producer is set up (hen/egg problem). As of this all configuration related messages are written directly to stdout.
Writing to a logfile that is not yet defined (and maybe never is) is a little bit tricky but there is already a similar mechanism in place that could be used - I'll look into that.

Loglevels are defined as follows: 
0 = only errors
1 = warnings + errors
2 = notes + warnings + errors
3 = debug + notes + warnings + errors

The start and stop messages are notes, so they are only printed with loglevel 2 or higher.

**Comment by arnecls** [comment]
fixed with 0.4.0
